Makefile

```diff
@@ -160,6 +160,11 @@ build-docker: require-APP build-docker-$(APP)
 build-docker-%: nix
 	@nix-build nix/docker.nix -A $(subst build-docker-,,$@) -o result-docker-$(subst build-docker-,,$@) --fallback
 
+test: require-APP
```

Comment:
there is a ``# TODO: py.test`` comment somewhere in ``nix/lib/default.nix``. i think you just need to uncomment the line and do ``py.test tests/``